src/Tribe/Views/V2/View.php

```diff
@@ -820,7 +839,14 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 
 		if ( ! empty( $passthru_vars ) && ! empty( $url ) ) {
 			// Re-apply the pass-thru query arguments.
-			$url = add_query_arg( $passthru_vars, $url );
+			$query_args = array_merge( $query_args, $passthru_vars );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 5f32cdd1e7eeba7f4d9dbb2af77c366a</sub>